src/cloudinary.js

```diff
@@ -730,6 +737,100 @@ class Cloudinary {
     return Transformation.new(this.config()).fromOptions(options).setParent(this);
   }
 
+  /**
+   *
+   * @param {HTMLElement} htmlElContainer
+   * @param {string} publicId
+   * @param {Object} options The {@link TransparentVideoOptions} options to apply.
+   */
+  createTransparentVideo(htmlElContainer, publicId, options = {}) {
+    return new Promise((resolveMainPromise, rejectMainPromise) => {
+
+      if (!htmlElContainer) {
+        rejectMainPromise({
+          status: 'error',
+          message: 'Expecting htmlElContainer to be HTMLElement'
+        });
+      }
+
+      options.autoplay = true;
+      options.muted = true;
+      options.controls = false;
+      options.max_timeout = options.max_timeout || 10000;
+      options.class = options.class || '';
+      options.class += ' cld-transparent-video';
+      options.seeThruURL = options.seeThruURL || 'https://unpkg.com/seethru@4/dist/seeThru.min.js';
```

Comment:
It's the only place we use the URL as a string, and I thought it's self explanatory, it's a matter of taste, I like how I don't need to look up what the default value is (even if it's a click). but if it's an issue it can be changed.